.github/workflows/ghcr.yml

```diff
@@ -2,7 +2,7 @@ name: Container image
 
 on:
   release:
-    types: [created]
+    types: [published]
```

Comment:
the pypi workflow also needs this change

Reply:
Done :ok_hand: